### ISSUE TITLE
Assign with no transition has `"ssign"` or `""` as target

### ISSUE THREAD

**Issue by yoav-lavi** [matthewp/liblucy#23]
Hey,
Noticed that if you try to make a transition containing only an assign action, it adds a `"ssign"` `target` if no other state is defined, or `""` if another state is defined (whereas I would expect no `target` in such a case)

```lucy
state initial {
  TEST => assign(someValue)
}
```

The resulting code is 

```js
import { createMachine, assign } from 'xstate';

export default createMachine({
  states: {
    initial: {
      on: {
        TEST: {
          target: 'ssign',
          actions: [
            assign({
              someValue: (context, event) => event.data
            })
          ]
        }
      }
    }
  }
});
```

And when doing the following:

```lucy
state initial {
  TEST => assign(someValue)
}

state other {}
```

The resulting code is 

```js
import { createMachine, assign } from 'xstate';

export default createMachine({
  states: {
    initial: {
      on: {
        TEST: {
          target: '',
          actions: [
            assign({
              someValue: (context, event) => event.data
            })
          ]
        }
      }
    },
    other: {

    }
  }
});
```

I'm aware that Lucy is in alpha and issues are to be expected, just wanted to give you a heads up :)

Good luck!

**Comment by matthewp** [comment]
Thanks! Yes this is indeed a bug. I'm curious if we should require a target state at the end of a transition or should we implicitly transition back into itself?

**Comment by matthewp** [comment]
This is fixed in `0.1.5`, thanks a lot for bringing the issue to my attention!
